CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -931,7 +972,11 @@ public let DECL_NODES: [Node] = [
       Child(
         name: "signature",
         kind: .node(kind: .functionSignature),
-        nameForDiagnostics: "function signature"
+        nameForDiagnostics: "function signature",
+        documentation: """
+          The arguments of the initializer.
+          While the function signature allows specifying a return clause, doing so is not semantically valid."
+          """
```

Comment:
Did you mix up functions and initializers here? 😉

Reply:
Yes, did some copy-pasting, is "A function signature that defines the interface of the function." better for this case, if not, need some help